releasenotes/notes/qpy-circuit-start-table-ac5d23c239bc4c68.yaml

```
@@ -0,0 +1,6 @@
+upgrade_qpy:
```

```suggestion
features_qpy:
```
The `upgrade` sections are actually intended only to let users know how they will need to upgrade their own code in order to work with the new version of Qiskit. Since introducing a new version of QPY isn't a breaking change, we should put it in `features_qpy` instead.

(If you were increasing our `QPY_COMPATIBILTY_VERSION` (which you are not), then we would want a note here to let users know they'd no longer be able to dump to some older version(s), for example.)